cmds/import_image/import_image_cmd.go

```diff
@@ -4,20 +4,47 @@ import (
 	"errors"
 	"fmt"
 
-	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	"github.com/cloudfoundry-community/bosh-softlayer-tools/common"
 
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	boshretry "github.com/cloudfoundry/bosh-utils/retrystrategy"
 	sldatatypes "github.com/maximilien/softlayer-go/data_types"
-	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	"github.com/maximilien/softlayer-go/softlayer"
+	"github.com/pivotal-golang/clock"
+	"time"
+)
+
+var (
+	locations = []sldatatypes.SoftLayer_Location{
```

Comment:
Seems array can't be declared as a constant, got the following error if I declare it a constant:
```
cmds/import_image/import_image_cmd.go:34: const initializer []data_types.SoftLayer_Location literal is not a constant
```
They should be regarded as initial locations where the public image will be stored, if a user needs to use the image in a different location, the image should be transferred to that location automatically. So I don't think it's necessary to add one more flag.